tests/test_schema.py

```diff
@@ -1633,6 +1677,22 @@ def test_validate_with_custom_error_handler(self):
 
 class TestFieldValidation:
 
+    @pytest.mark.parametrize('val', (None, False, 1, 1.2, object()))
+    def test_non_iterables_with_many(self, val):
```

Comment:
This test appears misplaced: it's not really about field validation, as the class name suggests. Perhaps this can just be a top-level test function.